packages/site/src/components/ContentCard.tsx

```diff
@@ -39,7 +39,7 @@ export const ContentCard = ({
             style={
               imageURL && theme === "dark"
                 ? {
-                    background: "var(--color-surface)",
+                    background: "var(--color-base-blue--200)",
```

Comment:
Arbitrary but this feels _just dark enough that it's not so glaring_ but without washing out the thumbnail contents

![image](https://github.com/user-attachments/assets/e63fb7c6-a6fc-4c8e-8a21-5314df4f8e3c)